integration/index.2.test.js

```diff
@@ -15,6 +18,58 @@ const {
   forceExitDaemon,
 } = require('@cnpmjs/rapid/lib/nydusd/nydusd_api');
 
+
+class Pids {
```

Comment:
We need to move this class into a separate file, perhaps under test/fixtures/util. It's better to keep one class per file for clarity and organization.